src/renderer/modals/BlacklistToken/index.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+
+import React, { useCallback } from "react";
+import { Trans, useTranslation } from "react-i18next";
+import ModalBody from "~/renderer/components/Modal/ModalBody";
+import Box from "~/renderer/components/Box";
+import Button from "~/renderer/components/Button";
+import Modal from "~/renderer/components/Modal";
+import { blacklistToken } from "~/renderer/actions/settings";
+import { useDispatch } from "react-redux";
+import Text from "~/renderer/components/Text";
+
+const BlacklistTokenModal = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const confirmBlacklistToken = useCallback(
+    tokenId => {
+      dispatch(blacklistToken(tokenId));
+    },
+    [dispatch],
+  );
+
+  return (
+    <Modal
+      name="MODAL_BLACKLIST_TOKEN"
+      centered
+      render={({ data, onClose }) => (
+        <ModalBody
+          preventBackdropClick
+          onClose={onClose}
+          title={t("blacklistToken.title")}
+          render={() => (
+            <Box>
```

Comment:
could also split out with a Body and a Footer 🤔

Reply:
sure